packages/cli/src/services/project.service.ee.ts

```diff
@@ -46,7 +47,8 @@ export class ProjectService {
 		private readonly projectRelationRepository: ProjectRelationRepository,
 		private readonly sharedCredentialsRepository: SharedCredentialsRepository,
 		private readonly cacheService: CacheService,
-		private readonly license: License,
+		private readonly licenseState: LicenseState,
+		private readonly globalConfig: GlobalConfig,
```

Comment:
Looks like we need only `DatabaseConfig` so we could pass in only that one instead as well.